application/src/main/scala/com/azavea/franklin/commands/Options.scala

```diff
@@ -1,9 +1,34 @@
 package com.azavea.franklin.api.commands
 
+import cats.data._
+import cats.effect._
+import cats.implicits._
+import com.azavea.franklin.crawler._
 import com.monovore.decline.Opts
 
 object Options extends DatabaseOptions with ApiOptions {
 
+  private val itemsFromCli: Opts[NonEmptyList[String]] = Opts.arguments[String]("items")
+
+  private def itemsFromFile[F[_]: Sync]: Opts[NonEmptyList[String]] =
+    Opts
+      .option[String]("file", "file containing URIs to STAC items to import", short = "f")
```

Comment:
I think it's worth calling out that this path needs to be absolute -- I tried with `items.csv` and `./items.csv` before `$(pwd)/items.csv`. I don't think I realized this was necessarily an absolute path when I did something similar in the s2 stac utils